src/components/studies/scheduler/Scheduler.tsx

```diff
@@ -827,7 +830,8 @@ const Scheduler: React.FunctionComponent<SchedulerProps> = ({
           </DialogButtonSecondary>
 
           <DialogButtonPrimary
-            disabled={!!!hasBursts}
+            disabled={!!!hasBursts && schedule.studyBursts?.length===0}
```

Comment:
why do we need this second check? can it be !!!hasBursts && schedule.studyBursts?.length>0? If so, there is probably a bug

Reply:
We can set schedule.studyBursts?.length < 1 instead of schedule.studyBursts?.length===0. As for the double check, the hasBursts is a boolean which is triggered by the toggle. Without the second check the user would not be able to save when they no longer want to do a burst.